classes/admin/class-submenu.php

```diff
@@ -555,7 +572,7 @@ protected function listTransactionsConvertExceptions($item)
             return $item;
         }
 
-        if (!empty($item->exceptions->hpsException)) {
+        if (!empty($item->exceptions->Exceptions)) {
```

Comment:
There's no direct mapping from the old SDK to the new one for this. We may have to reinvision how this operates with the new SDK.